code_pylint.py

```diff
@@ -35,7 +35,7 @@
                      'trailing-whitespace': 11,
                      'empty-docstring': 7,
                      'missing-module-docstring': 4,
-                     'too-many-arguments': 24,
+                     'too-many-arguments': 25,
```

Comment:
![image](https://github.com/Dessia-tech/plot_data/assets/42537234/064d3ff6-6ef1-4ef6-9a7f-352f4272bc56)

Reply:
Haha I just can agree and thank you for letting it for later